feature-proposals.rst

```diff
@@ -50,23 +89,28 @@ measured only since the formal discussion announcement as described above.
  Voting
 ********
 
-The author decides when it's time to move ahead and call a vote on the RFC. If
-the author feels that there's still healthy discussion going on, they can decide
-not to move ahead to request a vote after the minimal period, but extend it as
-needed. On the other hand, if they feel that the discussion is being derailed -
-they can always move ahead to a vote as long as the minimum discussion time
-elapsed.
-
-The vote is announced on the mailing list in a separate thread by sending an
-email with the subject ``[VOTE]``. It should reference the RFCs being voted on
-and if there are different options discussed, explain these options. It should
-also contain the URL of the page where the vote is taking place.
-
-Votes should be open for two weeks at minimum, at the authors discretion this
-may be extended, for example during holiday periods.
-
-A valid voting period must be declared when voting is started and must not be
-changed during the vote.
+RFC authors MAY start a vote after the minimum discussion period has elapsed.
+The intention of starting the vote MUST be announced at least 2 days (48 hours)
+before the start of the vote in the official discussion thread. RFC authors
+SHOULD NOT announce the start of the vote when the RFC discussion is still
+ongoing and new discussion points are brought forward. Similarly RFC authors
+SHOULD NOT proceed with an announced vote if new discussion points are brought
+forward after the voting announcement. If major or minor changes to the RFC text
+are made after announcing the vote, the discussion period MUST be extended and a
+new vote MUST be announced when the RFC is ready for voting after the new
+minimum discussion period has elapsed.
+
+The actual start of the vote MUST be announced on the mailing list in a separate
+thread with a ``[VOTE]`` prefix followed by the RFC title as the Subject. The
+email MUST include a link to the Wiki page of the RFC and to the mailing list
+archives of the discussion thread. It furthermore MUST clearly specify the
```

Comment:
That is https://news-web.php.net/php.internals ?

Reply:
Since the update to include the full threading view, https://news-web.php.net/php.internals would be the preferred link. But I would also be good with externals, since the IDs match up and folks are able to use either one. What's important to me is that folks are able to easily find the discussion.